pkg/antctl/command_definition.go

```diff
@@ -455,10 +467,10 @@ func (cd *commandDefinition) tableOutputForQueryEndpoint(obj interface{}, writer
 		// transform egress and ingress rules to string representation
 		egress, ingress := make([][]string, 0), make([][]string, 0)
 		for _, rule := range endpoint.Rules {
-			ruleStr := []string{rule.Name, rule.Namespace, strconv.Itoa(rule.RuleIndex), string(rule.UID)}
-			if rule.Direction == v1beta2.DirectionIn {
+			ruleStr := []string{rule.PolicyRef.Name, rule.PolicyRef.Namespace, strconv.Itoa(rule.RuleIndex), string(rule.PolicyRef.UID)}
+			if rule.Direction == controlplane.DirectionIn {
```

Comment:
Why change it to internal type? clients should use versioned type, internal types don't have guarantee compatibility.

Reply:
Reverted the change to v1beta2.